db/deploy/missing_metadata.sql

```diff
@@ -0,0 +1,77 @@
+-- Deploy ccbc:missing_metadata to pg
+
+begin;
+
+-- Table: ccbc_public.analyst
+comment on column ccbc_public.analyst.active is 'Boolean column indicating whether the analyst is active or not.';
+
+-- Table: ccbc_public.application_announcement
+comment on column ccbc_public.application_announcement.history_operation is 'Column describing the operation (created, updated, deleted) for context on the history page';
+
+-- Table: ccbc_public.application_gis_data
+comment on table ccbc_public.application_gis_data is 'Table containing the GIS data for the application';
+comment on column ccbc_public.application_gis_data.id is 'Primary key and unique identifier';
+comment on column ccbc_public.application_gis_data.application_id is 'The application_id of the application this record is associated with';
+comment on column ccbc_public.application_gis_data.batch_id is 'The batch_id is the unique identifier for the batch of gis data that was uploaded';
+comment on column ccbc_public.application_gis_data.json_data is 'The GIS data for the application in JSON format';
+
+-- Table: ccbc_public.application_package
+comment on column ccbc_public.application_package.application_id is 'The application_id of the application this record is associated with';
+comment on column ccbc_public.application_package.package is 'The package number the application is assigned to';
+
+-- Table: ccbc_public.application_status
+comment on column ccbc_public.application_status.status is 'The status of the application';
+
+-- Table: ccbc_public.application_status_type
+comment on column ccbc_public.application_status_type.visible_by_applicant is 'Boolean column used to differentiate internal/external status by indicating whether the status is visible to the applicant or not.';
+comment on column ccbc_public.application_status_type.visible_by_analyst is 'Boolean column used to differentiate internal/external status by indicating whether the status is visible to the analyst or not.';
+comment on column ccbc_public.application_status_type.status_order is 'The logical order in which the status should be displayed.';
+
+-- Table: ccbc_public.assessment_data
+comment on table ccbc_public.assessment_data is 'Table containing the assessment data for the application';
+comment on column ccbc_public.assessment_data.id is 'Primary key and unique identifier';
+comment on column ccbc_public.assessment_data.application_id is 'The application_id of the application this record is associated with';
+comment on column ccbc_public.assessment_data.json_data is 'The assessment data in JSON format';
```

Comment:
Removerino https://github.com/bcgov/CONN-CCBC-portal/pull/1806

Reply:
Removed all of these